safetensors/src/tensor.rs

```diff
@@ -41,6 +43,12 @@ impl From<std::io::Error> for SafeTensorError {
     }
 }
 
+impl From<serde_json::Error> for SafeTensorError {
+    fn from(error: serde_json::Error) -> SafeTensorError {
+        SafeTensorError::JsonError(error)
+    }
+}
+
```

Comment:
please let me know if you'd like me to submit `thiserror` PR so that we don't have to manually write error conversions